src/containerapp/azext_containerapp/_utils.py

```diff
@@ -726,3 +727,17 @@ def parse_build_env_vars(env_list):
         })
 
     return env_var_def
+
+
+def convert_egress_parameter(egress_enabled):
+    if egress_enabled:
+        return "egressEnabled"
+    else:
+        return "egressDisabled"
+
+
+class AppType(Enum):
```

Comment:
Where did you use this Enum?

Reply:
Here `validate_container_app_name(self.get_argument_name(), AppType.SessionPool.name)`